src/AWSBatch.jl

```diff
@@ -1,18 +1,19 @@
 module AWSBatch
 
+using AWS
 using AutoHashEquals
-using AWSCore: AWSException, aws_config
-using AWSSDK.Batch
-using AWSSDK.CloudWatchLogs
-using AWSTools.EC2: instance_region
 using OrderedCollections: OrderedDict
 using Dates
 using Memento
 using Mocking
 
+@service Batch
+@service Cloudwatch_Logs
```

Comment:
A bit nit-picky, but to be consistent with how AWS refers to the service.

```suggestion
@service CloudWatch_Logs
```

Reply:
It's not somehow case insensitive is it?  The [AWS.jl docs](https://juliacloud.github.io/AWS.jl/stable/services/cloudwatch_logs.html) refer to this module this way.

Reply:
The docs are a bit messy. I believe they're set this way because our source for defining them calls it `cloudwatch_logs` in all lowercase and we try to tidy them up a bit. They're probably slightly off, as AWS refers to it as [CloudWatch](https://aws.amazon.com/cloudwatch/).